data/autoyast_xen/sles12sp4PV_PRG.xml

```diff
@@ -1132,55 +1066,12 @@ DefaultPolicy default
     <install_recommended config:type="boolean">true</install_recommended>
     <instsource/>
     <packages config:type="list">
-      <package>sudo</package>
```

Comment:
I believe we still need some of the basic packages. I would leave at least the following in here:

* `sudo`
* `binutils`
* `iputils`

In addition I think changes on the packages list requires a full VR.

Reply:
Yes :D We need all of those.

Reply:
Yes, these are basic packages. However, we already specify minimal and base patterns in profile. These packages are included in patterns so they will be installed. We don't need to explicitly specify them again in packages list. 

Yes, a full VR makes sense.

Reply:
Ahhh, I understand, now it makes sense. If the VR is passing, then this is good!